projects/kit/components/combo-box/combo-box.directive.ts

```diff
@@ -134,8 +134,10 @@ export class TuiComboBox<T>
     }
 
     public override writeValue(value: T | string | null): void {
-        super.writeValue(value);
-        this.textfield.value.set(this.stringify(value));
+        if (value !== this.value()) {
+            super.writeValue(value);
+            this.textfield.value.set(this.stringify(value));
```

Comment:
But what is about `formControl.reset` ?

Reply:
- https://github.com/taiga-family/taiga-ui/pull/11366

**TODO**: rebase current branch onto `main` after its merge